PingPayments.PaymentsApi.Tests/V1/PaymentOrderResourceTests.cs

```diff
@@ -27,18 +28,37 @@ public async Task Get_404_on_non_existing_order()
             AssertHttpNotFound(response);
         }
 
+
         [Fact]
-        public async Task Can_create_order_without_split_tree_id()
+        public async Task Can_create_order_with_split_parameters()
         {
-            var request = new CreatePaymentOrderRequest(CurrencyEnum.SEK);
+            var splitParamters = new Dictionary<string, object> { { "tenant_fee", 20.ToMinorCurrencyUnit() } }; 
```

Comment:
Ser bra ut, möjligtvis att man skulle använda ett dynamic object istället för en dictionary, lite mer "tillgängligt".
Då skulle du istället kunna skriva `dynamic splitParamters = new { tenant_fee = 20.ToMinorCurrencyUnit() };`

Reply:
Men du gör som du vill, egentligen funkar en dictionary lika bra

Reply:
Din lösning är helt klart snyggare!

Jag behövde dock lägga till "member name" för att det skulle funka: 
`dynamic splitParamters = new { fee = "tenant_fee", amount=20.ToMinorCurrencyUnit() };` 
utan dessa fick jag:
`Invalid anonymous type member declarator. Anonymous type members must be declared with a member assignment, simple name or member access.`

Reply:
Din lösning är helt klar snyggare! Jag kör på den!